app/assets/javascripts/components/jquery_wrappers/Popover.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component, Fragment } from 'react';
+
+export default class Popover extends Component {
+  constructor(props) {
+    super(props);
+
+    this.saveChildRef = this.saveChildRef.bind(this);
+    this.saveContentRef = this.saveContentRef.bind(this);
+  }
+
+  componentDidMount() {
+    const {
+      delay,
+      trigger,
+      title
+    } = this.props;
+
+    $(this.childRef).popover({
+      delay,
+      trigger,
+      title,
+      html: true,
+      content: this.contentRef
+    });
+  }
+
+  componentWillUnmount() {
+    $(this.childRef).popover('destroy');
+  }
+
+  saveChildRef(ref) {
+    this.childRef = ref;
+  }
+
+  saveContentRef(ref) {
+    this.contentRef = ref;
+  }
+
+  render() {
+    return (
+      <div>
+        { this.props.children({ ref: this.saveChildRef }) }
+        <div style={{ display: 'none' }}>
```

Comment:
Is it not better to move this style to a class?

Reply:
I don't think so, this div is just to hide its content. I don't believe that it is worthy of going to a style class since is just too small.